rest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * REST APIs for ajax responder for the search system
+ */
+
+namespace Clone_Replace;
+
+/**
+ * A function to register custom REST routes on REST API init.
+ */
+function action_rest_api_init() {
+	register_rest_route(
+		'clone-replace/v1',
+		'/search',
+		[
+			'callback'            => __NAMESPACE__ . '\rest_route_search',
+			'methods'             => 'GET',
+			'permission_callback' => '__return_true',
```

Comment:
We should probably make sure that there is a logged in user, at least, so this endpoint can't be hit from the front end by unauthenticated users.